### PR TITLE
101 unmap pinning rule

### PR DESCRIPTION
**Pull request by PatrykSzwer**
#101

### CODE REVIEW

File: src/WebsiteProvider/Api/MappingHandlers.cs [hunk 1]
```diff
@@ -54,13 +55,24 @@ public void MapPinningRule(WebMap webMap, string registeredSectionUri = null)
             Blender.MapUri(webMap.ForeignUrl, token);
         }
 
+        public void UpdatePinningRule(WebMap webMap)
+        {
+            var newUri = webMap.ForeignUrl;
+            webMap.ForeignUrl = GetOldUri(webMap);
```
Comment (un-tone) [inline src/WebsiteProvider/Api/MappingHandlers.cs]: From my point of view it is not a good practice to make changes in DB objects just to pass some value to a method. In current case I recommend to add additional parameter, e.g. `string webMapForeignUrl = null` and use it in called methods like this `webMapForeignUrl ?? webMap.ForeignUrl`.

Comment (PatrykSzwer) [inline src/WebsiteProvider/Api/MappingHandlers.cs]: That's actually the idea I was looking for! Thanks!

File: src/WebsiteProvider/Api/MappingHandlers.cs [hunk 2]
```diff
@@ -54,17 +55,25 @@ public void MapPinningRule(WebMap webMap, string registeredSectionUri = null)
...
-        public void UnmapPinningRule(WebMap webMap)
+        public void UpdatePinningRule(WebMap webMap)
         {
+            this.UnmapPinningRule(webMap, GetOldUri(webMap));
+            this.MapPinningRule(webMap);
+        }
+
+        public void UnmapPinningRule(WebMap webMap, string webMapForeignUrl = null)
+        {
+            webMapForeignUrl = webMapForeignUrl ?? webMap.ForeignUrl;
```
Comment (un-tone) [inline src/WebsiteProvider/Api/MappingHandlers.cs]: A program should not do anything redundant. This line should be moved below after the condition block.

Comment (PatrykSzwer) [inline src/WebsiteProvider/Api/MappingHandlers.cs]: Done

**Comment by un-tone** [comment]
Please don't forget to create a new issue for tests covering this functionality.

**Comment by PatrykSzwer** [comment]
I've created an issue for tests here: https://github.com/StarcounterApps/Website/issues/126

**Pull request merged**